src/main/java/org/zalando/nakadi/controller/ExceptionHandling.java

```diff
@@ -85,4 +87,18 @@ private String generateErrorTraceId() {
         }
         throw exception.getException();
     }
+
+    @ExceptionHandler(RepositoryProblemException.class)
+    public ResponseEntity<Problem> handleRepositoryProblem(final RepositoryProblemException exception,
+                                                           final NativeWebRequest request) {
+        LOG.error("Repository problem occurred", exception);
+        return Responses.create(Response.Status.SERVICE_UNAVAILABLE, exception.getMessage(), request);
+    }
+
+    @ExceptionHandler(InconsistentStateException.class)
```

Comment:
@adyach I think that we can just reuse this method for all other 500-resulting exceptions by just adding exception class to `@ExceptionHandler` annotation. (And of course we will change parameter type to MyNakadiRuntimeException1)